Framework/Algorithms/src/Rebin.cpp

```diff
@@ -89,31 +139,30 @@ void Rebin::init() {
 
   declareProperty(std::make_unique<ArrayProperty<double>>("Params", std::make_shared<RebinParamsValidator>()),
                   "A comma separated list of first bin boundary, width, last bin boundary. "
-                  "Optionally "
-                  "this can be followed by a comma and more widths and last boundary "
-                  "pairs. "
-                  "Optionally this can also be a single number, which is the bin width. "
-                  "In this case, the boundary of binning will be determined by minimum and "
-                  "maximum TOF "
-                  "values among all events, or previous binning boundary, in case of event "
-                  "Workspace, or "
-                  "non-event Workspace, respectively. Negative width values indicate "
-                  "logarithmic binning. ");
+                  "Optionally this can be followed by a comma and more widths and last boundary pairs. "
+                  "Optionally this can also be a single number, which is the bin width. In this case, the boundary of "
+                  "binning will be determined by minimum and maximum TOF values among all events, or previous binning "
+                  "boundary, in case of event Workspace, or non-event Workspace, respectively. "
+                  "Negative width values indicate logarithmic binning.");
 
   declareProperty("PreserveEvents", true,
-                  "Keep the output workspace as an EventWorkspace, "
-                  "if the input has events. If the input and output EventWorkspace "
-                  "names are the same, only the X bins are set, which is very quick. If "
-                  "false, "
-                  "then the workspace gets converted to a Workspace2D histogram.");
+                  "Keep the output workspace as an EventWorkspace, if the input has events. If the input and output "
+                  "EventWorkspace names are the same, only the X bins are set, which is very quick. If false, then the "
+                  "workspace gets converted to a Workspace2D histogram.");
 
-  declareProperty("FullBinsOnly", false, "Omit the final bin if it's width is smaller than the step size");
+  declareProperty("FullBinsOnly", false, "Omit the final bin if its width is smaller than the step size");
 
   declareProperty("IgnoreBinErrors", false,
-                  "Ignore errors related to "
-                  "zero/negative bin widths in "
-                  "input/output workspaces. When ignored, the signal and "
-                  "errors are set to zero");
+                  "Ignore errors related to zero/negative bin widths in input/output workspaces. When ignored, the "
+                  "signal and errors are set to zero");
+
+  declareProperty("UseReverseLogarithmic", false,
+                  "For logarithmic intervals, the splitting starts from the end and go back to the start, ie the bins "
```

Comment:
There are spelling mistakes here:
```suggestion
                  "For logarithmic intervals, the splitting starts from the end and goes back to the start, ie the bins "
```